tor_core/helpers.py

```diff
@@ -1,12 +1,25 @@
 import logging
 import re
 import sys
+import time
 
 import prawcore
 import requests
 
 from tor_core import __version__
 from tor_core.strings import bot_footer
+from tor_core.config import config
+
+subreddit_regex = re.compile(
+    'reddit.com\/r\/([a-z0-9\-\_\+]+)',
+    flags=re.IGNORECASE
+)
+
+default_exceptions = (
```

Comment:
Because there are a lot more exceptions from praw that we're deliberately not catching, like loginerror. These are just the ones that we decided needed to be the defaults for keeping the bot running under normal conditions.